resources/lib/sutils.py

```diff
@@ -131,23 +131,27 @@ def scanRunning(self):
         return (xbmc.getCondVisibility('Library.IsScanningVideo') or
                 xbmc.getCondVisibility('Library.IsScanningMusic'))
 
-    def getBBDB(self, name):
-        name = util.request('http://csfd.bbaron.sk/find.php?' +
-                            urllib.urlencode({'sosac': 1, 'name': name}))
-        if name != '':
-            return self.getTVDB(name, 1)
-        return None
-
-    def getTVDB(self, name, level=0):
-        data = util.request('http://thetvdb.com/api/GetSeries.php?' +
-                            urllib.urlencode({'seriesname': name, 'language': 'cs'}))
-        try:
-            tvid = re.search('<id>(\d+)</id>', data).group(1)
-        except:
-            if level == 0:
-                tvid = self.getBBDB(name)
-            else:
+    def getTVDB(self, name, id):
+        if id:
+            data = (util.request('http://thetvdb.com/api/GetSeriesByRemoteID.php?imdbid=tt' +
+                    id + '&language=all'))
+            try:
+                tvid = re.search('<id>(\d+)</id>', data).group(1)
```

Comment:
Ze uz nejaky kod uz bol neznamena, ze je korektny. Treba sa preto vzdy nad tym zamysliet. Parsovanie XML nie je python-specific vec, takze to celkom neberiem. Ak chces vediet o parsovani XML v pythone viac, prozri sa [sem](https://docs.python.org/2/library/xml.etree.elementtree.html).